src/users/containers/users/users.component.ts

```diff
@@ -22,37 +22,40 @@ export class UsersComponent implements OnInit, OnDestroy {
   public currentPageNumber: number = 1;
   public pageTotalSize: number;
   public allUsersList$: Subscription;
+  public filterValues: string = '';
+  public userList: Array<object>;
+
+  public searchFiltersEnabled$: Observable<boolean>;
 
   constructor(
     private readonly store: Store<fromStore.UserState>,
-    private readonly usersService: UsersService
+    public readonly featureToggleService: FeatureToggleService
   ) {}
 
   public ngOnInit(): void {
-    // Call to usersService.getAllUsersList() is required to set pageTotalSize for pagination purposes
-    this.allUsersList$ = this.getAllUsers();
-    this.loadUsers(this.currentPageNumber - 1);
+    this.loadUsers();
+    // TODO: ensure feature toggle name is added.
```

Comment:
I think the feature toggle code is merged in. You can use that now

Reply:
Added flag